src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java

```diff
@@ -31,7 +33,7 @@
     public Patient mapToPatient(final Message message) {
 
         final Optional<InvestigationSubject> investigationSubject =
-            Optional.ofNullable(message.getServiceReportDetails())
+            ofNullable(message.getServiceReportDetails())
```

Comment:
`ofNullable` on its own doesn't mean much; better to keep the `Optional` here imo; what do you think?

Reply:
Sure, makes sense. Thanks!
4c4a042dca82dd539b67888b275c2a4b3c329293